### ISSUE TITLE
Album covers visibility 

### ISSUE THREAD

**Issue by RadiantBrr** [basharovV/musicat#30]
Album covers are not showing up despite of it being in the metadata of the music file,
![image](https://github.com/user-attachments/assets/b96db3a4-3c6b-4bba-b49b-4ac8a5e37c6f)
Shown below:
![image](https://github.com/user-attachments/assets/4a0e7049-a7de-494e-a09b-5cf53156baf1)
[toggle to turn off the invisible background would be neat :)]
![image](https://github.com/user-attachments/assets/5d955ad4-0d73-4ce3-9229-19bc4c4ea832)

Adding to the above, "Fetch art" yields 'Couldn't find album art'.
Platform: Windows

**Comment by daiyam** [comment]
Yep, the tags are missing while using the `release` version. In `dev`, the tags are correctly displayed...

I also have the artworks disappearing on me. I will try to find a way to reproduce the issue.

**Comment by basharovV** [comment]
I think the empty artworks in album view should be fixed in https://github.com/basharovV/musicat/commit/7cd2702e56898d9b863a6956a3f7b4ee9ac69242 - was caused by a bug reading the settings file. It's not yet released though, I'll try and get a new release in the next couple weeks.

**Comment by basharovV** [comment]
Fixed in b0711858c352246d8ec425f7e2f82229ffd386e4 (applied blur to the hinge), and also adjusted the album info so it doesn't overlap with other albums. There's also a new shimmer effect when playing.
